spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.shared_context 'when authenticated User' do
+  let(:user) { create(:user) }
```

Comment:
I prefer to name this variable `current_user` instead of `user` because it seems clearer to me read current_user as an authenticated user

Reply:
Done